### ISSUE TITLE
'100 arrays dict' fails

### ISSUE THREAD

**Issue by RSabet** [mre/hyperjson#20]
using the benchmarks from [rapidjson](https://github.com/lelit/python-rapidjson/blob/master/benchmarks/test_benchmark.py)  `100 arrays dict` fails with hyperjson.

**Comment by wdv4758h** [comment]
This looks like a small case that which crash on my end.

```
import hyperjson                 
                                 
hyperjson.dumps([{'a': 0}]*10000)
```

**Comment by mre** [comment]
Just tried again on my machine and it works now in master, most likely due to fixes in pyo3 0.5. 🎉 
@konstin fyi.

**Comment by konstin** [comment]
I hope this has been fixed by 0.5.0 and is not another instance of https://github.com/PyO3/pyo3/issues/271, which is tackled in https://github.com/PyO3/pyo3/pull/281

**Comment by mre** [comment]
At least the output is as expected:

```
>>> hyperjson.dumps([{'a': 0}]*10000)
'[{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},{"a":0},...,{"a":0},{"a":0}]'
```
